third_party/forked/glog/glog_file.go

```diff
@@ -36,15 +35,17 @@ var MaxSize uint64 = 1024 * 1024 * 1800
 // logDirs lists the candidate directories for new log files.
 var logDirs []string
 
-// If non-empty, overrides the choice of directory in which to write logs.
-// See createLogDirs for the full list of possible destinations.
-var logDir = flag.String("log_dir", "", "If non-empty, write log files in this directory")
 
 func createLogDirs() {
-	if *logDir != "" {
-		logDirs = append(logDirs, *logDir)
+	// if log-file is not specified, only then try to use logDir
+	if logging.logFile == "" {
+		// if log-dir is not specified, then use temp directory
```

Comment:
This is a pretty deep change - it changes "log to all dirs" into "log to one dir", if I read correctly.  Why bother to keep logDirs as a plural at this point?